docker-compose.yaml

```
@@ -1,15 +1,32 @@
 version: "3.8"
 services:
+  postgres:
+    image: postgres:13-alpine
+    ports:
+      - "127.0.0.1:7777:5432"
```

```suggestion
      - "7777:5432"
```
Pretty sure this isn't a valid port mapping.